PSMultiLog.psm1

```diff
@@ -1288,24 +1297,42 @@ Function Write-HostLog {
     )
 
```

Comment:
Could you add a `Begin {}` block up here and define all those constant variables in it (`$SGRESC`, `$CLEAR`, `$RED`, `$YELLOW`, and `$CYAN`)? They'll be in scope in the `Process {}` block. Functionally the same, but just a bit cleaner.

Reply:
Sure! it  is a good idea. I will push my change  later.

Reply:
> Sure! it is a good idea. I will push my change later.

Pushed.